resources/phpcs.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
```

Comment:
Shouldn't we provide the same config as for the demo shop?
Like https://github.com/spryker-shop/b2b-demo-shop/blob/master/phpcs.xml